paddle/fluid/inference/api/demo_ci/run.sh

```diff
@@ -68,6 +83,10 @@ rm -rf *
 
 for WITH_STATIC_LIB in ON OFF; do
   if [ $(echo `uname` | grep "Win") != "" ]; then
+    # TODO(wilber, T8T9): Do we still need to support windows gpu static library
```

Comment:
whether static lib is needed in CPU?

Reply:
The cpu library is very small, it needs to be evaluated in the future whether to keep.